test/integrationtest/projecttest/main_test.go

```diff
@@ -0,0 +1,19 @@
+// +build integration
+
+package projecttest
+
+import (
+	"os"
+	"testing"
+	//"time"
+)
+
+//TestMain - golang TestMain - used for setup and teardown
+func TestMain(m *testing.M) {
+	deferfunc, _ := BootupDemoProject()
+	result := m.Run()
+	deferfunc()
```

Comment:
```suggestion
	defer deferfunc()
```
before `m.Run()` in case something bad happens?

Reply:
will not work because then its called after the os.Exit